engine/class_modules/sc_death_knight.cpp

```diff
@@ -856,6 +859,7 @@ struct death_knight_t : public player_t {
   {
     double lucid_dreams_minor_proc_chance = 0.15;
     bool disable_aotd = false;
+    bool killing_machine_rppm = false;
```

Comment:
I don't think the rppm killing machine options and override are necessary going into shadowlands, it was mostly just a testing tool to compare how off it was compared to the new formula.

Reply:
Perfect, I'll pull those sections out.